tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['*.html'],
+  prefix: 'tw-',
```

Comment:
I like the `tw-` prefix :) very helpful to differentiate custom classes from standard tw classes